src/app/forms/components/number-input-with-suffix/number-input-with-suffix.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="govuk-form-group">
+  <h1 class="govuk-label-wrapper">
+    <label class="govuk-label govuk-label--m" for="{{ name }}">{{ label }}</label>
+  </h1>
+
+  <app-field-error-message [error]="error" [name]="name"></app-field-error-message>
+
+  <div *ngIf="isEditing; else view" class="govuk-input__wrapper">
+    <input
+      class="govuk-input govuk-input--width-3"
+      id="{{ name }}"
+      name="{{ name }}"
+      type="number"
+      [(ngModel)]="value"
+      [disabled]="disabled"
+      (ngModelChange)="onChange($event)"
+      (blur)="onTouched(); handleEvent($event)"
+      (focus)="handleEvent($event)"
+      appNumberOnly
+    />
+    <div class="govuk-input__suffix" aria-hidden="true">{{ suffix }}</div>
+  </div>
```

Comment:
why not upgrade the existing number input component to accept child content for suffix.
```suggestion
  <div *ngIf="isEditing; else view" class="govuk-input__wrapper">
    <input
      class="govuk-input govuk-input--width-3"
      id="{{ name }}"
      name="{{ name }}"
      type="number"
      [(ngModel)]="value"
      [disabled]="disabled"
      (ngModelChange)="onChange($event)"
      (blur)="onTouched(); handleEvent($event)"
      (focus)="handleEvent($event)"
      appNumberOnly
    />
    <div class="govuk-input__suffix" aria-hidden="true">
      <ng-content select="[suffix]"></ng-content>
    </div>
  </div>
```

This can then be used as:
```
<app-number-input>
  <ng-container suffix> % </ng-container>
</app-number-input>
```

Reply:
I get what you mean but the width and styling of the 2 are different. It will require more testing.

I still have some tickets revolving around the new service designs. I will try to merge the 2 in a future PR, as this one is already very bloated.